CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 # basic setup for cmake
-cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
```

Comment:
I don' think this is too restrictive these days. @maddyscientist is using 3.20rc features